PixivImage.py

```diff
@@ -221,7 +221,7 @@ def ParseInfo(self, page):
         if self._tzInfo is not None:
             self.worksDateDateTime = self.worksDateDateTime.astimezone(self._tzInfo)
 
-        tempDateFormat = self.dateFormat or "%m/%d/%y %H:%M"  # 2/27/2018 12:31
+        tempDateFormat = self.dateFormat or "%Y.%m.%d %H:%M"  # 2018.2.27 12:31
```

Comment:
I believe if the date format is going to change, it should be changed to conform to ISO8601.
https://en.wikipedia.org/wiki/ISO_8601
There could be multiple ways to format it though..
- Use dashes, or no separators between the dates
- Use a `T` or a space to separate the date from the time, or do not separate them at all
- Use a colon to separate hours from minutes, or do not separate them at all
```
        tempDateFormat = self.dateFormat or "%Y%m%d %H:%M"    # 20182207 12:31
        tempDateFormat = self.dateFormat or "%Y%m%d %H%M"     # 20182207 1231
        tempDateFormat = self.dateFormat or "%Y%m%dT%H:%M"    # 20182207T12:31
        tempDateFormat = self.dateFormat or "%Y%m%dT%H%M"     # 20182207T1231
        tempDateFormat = self.dateFormat or "%Y-%m-%d %H:%M"  # 2018-22-07 12:31
        tempDateFormat = self.dateFormat or "%Y-%m-%d %H%M"   # 2018-22-07 1231
        tempDateFormat = self.dateFormat or "%Y-%m-%dT%H:%M"  # 2018-22-07T12:31
        tempDateFormat = self.dateFormat or "%Y-%m-%dT%H%M"   # 2018-22-07T1231
```
Of this list, I would suggest going with this:
```suggestion
        tempDateFormat = self.dateFormat or "%Y%m%dT%H%M"     # 20182207T1231
```

Reply:
Oh, I wasn't aware that ISO_8601 extended also to the separators of the values, thank you for pointing it out!
As for which format to go with we could also go with just what Pixiv's API returns us directly, giving us something like "2019-08-30T14:51:40+00:00", although I think a shorter solution like yours is better (and colons wouldn't work on Windows)